src/ucp/core/ucp_context.c

```diff
@@ -1785,3 +1785,20 @@ uint64_t ucp_context_dev_idx_tl_bitmap(ucp_context_h context,
 
     return tl_bitmap;
 }
+
+const char* ucp_context_cm_name(ucp_context_h context,
+                                ucp_rsc_index_t cm_idx, char *name,
```

Comment:
according to the usage you can pass cmpt_idx directly. This way you will not need to find it here

Reply:
this function is used in other places too
sorry, can't do it